CHANGELOG.md

```diff
@@ -363,6 +363,11 @@ This is a corrective release for [2.1.1].
   replacing the material that was previously baked into `sigstore._store`
   ([#351](https://github.com/sigstore/sigstore-python/pull/351))
 
+### Changed
+* `sigstore verify and sign` doesn't support `--certificate-chain` and `--rekor-root-pubkey`
```

Comment:
This should be under `### Removed`, I think 🙂

Reply:
Right! Changing now!